users/static/users/scripts/jquery.fittext.js

```diff
@@ -0,0 +1,43 @@
+/*global jQuery */
```

Comment:
Can't we use this script from link instead of copying it to our code?

Reply:
I've forgotten about this static file. It's not used anymore for scaling text so I'll just remove it.